src/h5web/toolbar/controls/DomainSlider/ScaledSlider.tsx

```diff
@@ -91,13 +80,14 @@ function ScaledSlider(props: Props): ReactElement {
       max={SLIDER_RANGE[1]}
       value={scaledValue}
       disabled={disabled}
+      onBeforeChange={(bounds) => setBeforeChangeValue(bounds as Domain)}
       onChange={(bounds) => handleChange(bounds as Domain)}
-      onAfterChange={handleAfterChange}
+      onAfterChange={(bounds) => handleChange(bounds as Domain, true)}
```

Comment:
I can now reuse the same function as `onChange` to avoid storing `hasMinChange` and `hasMaxChanged` in local states.